flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarRunHandler.java

```diff
@@ -130,24 +130,33 @@ private SavepointRestoreSettings getSavepointRestoreSettings(
 
         final JarRunRequestBody requestBody = request.getRequestBody();
 
+        Configuration effectiveConfiguration = requestBody.getFlinkConfiguration();
```

Comment:
```suggestion
        final Configuration configuration = requestBody.getFlinkConfiguration();
```
Just renaming this to `effectiveConfiguration` doesn't achieve anything. It's misleading even because it's _not_ the effective configuration.